.github/workflows/package-release.yml

```diff
@@ -0,0 +1,42 @@
+name: Package release
+
+on:
+  pull_request:
+    types: 
+      - closed
+    branches: 
+      - main
+
+jobs:
+  package-release:
+    runs-on: ubuntu-latest
+    if: |
+      startsWith(github.head_ref, 'release-'')
+      && github.event.pull_request.merged
+      
+    steps: 
+      - name: Checkout out repository
+        uses: actions/checkout@v2
+
+      - name: Set up Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: '12.x'
+          cache: 'yarn'
+      
+      # The yarn cache is not node_modules
+      - name: Install Dependencies
+        run: yarn --prefer-offline
+      
+      - name: Run tests
+        run: yarn test
+
+      - name: Package library
+        run: yarn pack
```

Comment:
After @suzubara's demo today, I have questions. Do I need to explicitly run `yarn build` instead of just install for this?

`yarn install` and `yarn build` (which just runs `webpack --progress`) both create the `lib` directory, and I'm having trouble understanding and finding info about what the difference between the two is besides the --progress flag.